app/views/submissions/steps/_visibility.html.haml

```diff
@@ -4,8 +4,9 @@
     contact with support team.
   %p
     By default sending your submission will make provided data
-    %strong publicly available.
-    You can keep your data private by changing
+    %strong private.
+
+    You can make your data publicly available by changing
     %strong visibility status
     below.
```

Comment:
One suggestion - add a remark that data submitted as private could be made public at any time later.

Reply:
Done.

Reply:
Is it?

Reply:
I forgot to push the commit before merging and then added it directly to master (41f0ee5834f7b64e390fa1d15b86e47113f929ff).

Reply:
All right.